### PR TITLE
Update air-interpreter to 0.11.0

### PR DESCRIPTION
**Pull request by folex**

### CODE REVIEW

File: aquamarine/src/plumber.rs [hunk 1]
```diff
@@ -120,6 +123,14 @@ impl<RT: AquaRuntime> Plumber<RT> {
                     }
                     ActorPoll::Executing => {}
                 }
+            } else {
+                if mailbox_size > 11 {
```
Comment (justprosh) [inline aquamarine/src/plumber.rs:127]: Just a number looks strange to me. Need a comment or smth

File: particle-node/tests/builtin.rs [hunk 2]
```diff
@@ -36,6 +36,7 @@ use libp2p::kad::kbucket::Key;
 use eyre::WrapErr;
 use itertools::Itertools;
 use libp2p::PeerId;
+use log_utils::enable_logs;
```
Comment (justprosh) [inline particle-node/tests/builtin.rs:39]: Plz remove debug import

File: particle-node/tests/loop_topology.rs [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+use created_swarm::{make_swarms, CreatedSwarm};
+use test_constants::KAD_TIMEOUT;
+use test_utils::{create_service, timeout};
+
+use eyre::{ContextCompat, WrapErr};
+use futures::executor::block_on;
+use itertools::Itertools;
+use libp2p::core::Multiaddr;
+use local_vm::read_args;
+use log_utils::enable_logs;
```
Comment (justprosh) [inline particle-node/tests/loop_topology.rs:27]: ditto

File: particle-node/tests/network_explore.rs [hunk 4]
```diff
@@ -23,6 +23,7 @@ use futures::executor::block_on;
 use itertools::Itertools;
 use libp2p::core::Multiaddr;
 use local_vm::read_args;
+use log_utils::enable_logs;
```
Comment (justprosh) [inline particle-node/tests/network_explore.rs:26]: ditto

File: aquamarine/src/plumber.rs (hunk 1 above)
Comment (folex) [reply aquamarine/src/plumber.rs:128]: It's just for debugging purposes. Ideally, it should be something dynamic looking for abnormalities. I'll add TODO cuz currently there's not much can be done with that number.

**Review by justprosh — APPROVED** [review]

**Pull request merged**